cmd/util/cmd/verify_execution_result/cmd.go

```diff
@@ -56,7 +58,21 @@ func init() {
 
 	Cmd.Flags().BoolVar(&flagStopOnMismatch, "stop_on_mismatch", false, "stop verification on first mismatch")
 
-	Cmd.Flags().BoolVar(&flagtransactionFeesDisabled, "fees_disabled", false, "disable transaction fees")
+	Cmd.Flags().BoolVar(&flagTransactionFeesDisabled, "fees_disabled", false, "disable transaction fees")
+
+	Cmd.Flags().BoolVar(
+		&flagVMScriptExecutionEnabled,
+		"vm_script_execution_enabled",
+		false,
+		"enable script execution with VM",
+	)
+
+	Cmd.Flags().BoolVar(
+		&flagVMTransactionExecutionEnabled,
+		"vm_transaction_execution_enabled",
+		false,
+		"enable transaction execution with VM",
+	)
```

Comment:
@j1010001 is this what you were asking for, support for running scripts and transactions with the VM in the execution verification tool?

Reply:
yup, thanks!